auth/jwt/token_test.go

```diff
@@ -148,6 +164,12 @@ func TestParse(t *testing.T) {
 			token: newToken(issuerName, key()),
 			err:   authjwt.ErrJSONHandle,
 		},
+		{
```

Comment:
Add the following cases
- parse token with an empty domain
- parse token with empty subject